src/App.tsx

```diff
@@ -1,204 +1,54 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { ErrorNotifications } from './components/Errors';
+
+import { Todo } from './types/Todo';
+import { Errors } from './types/ErrorsEnum';
+import { Filter } from './types/FilterEnum';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(Filter.ALL);
+  const [errorNotification, setErrorNotification] = useState<Errors>(
+    Errors.DEFAULT,
+  );
+
+  useEffect(() => {
+    setErrorNotification(Errors.DEFAULT);
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorNotification(Errors.LOADING);
+      });
+    setTimeout(() => {
+      setErrorNotification(Errors.DEFAULT);
+    }, 3000);
```

Comment:
You should do it in any error case, you can create a handler to set and remove errors so you can reuse it
```typescript
const handleError = (error: Errors) => {
  setErrorMessage(error);

  ssetTimeout(() => {
      setErrorNotification(Errors.DEFAULT);
    }, 3000);
};
```